testdata/p4_16_bmv_errors/emit-struct-bmv2.p4

```diff
@@ -0,0 +1,98 @@
+/*
+    Copyright 2018 MNK Consulting, LLC.
+    http://mnkcg.com
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+//  This test fails for bmv2 backend because the backend does not support
+//  a struct arg with emit().
+//
+#include <v1model.p4>
+
+struct alt_t {
+  bit<1> valid;
+  bit<7> port;
+};
+
+@MNK_annotation("(test flatten)")
+struct row_t {
+  alt_t alt0;
+  alt_t alt1;
+};
+
+header bitvec_hdr {
+  row_t row;
+};
+
+struct col_t {
+  bitvec_hdr bvh;
+};
+
+struct local_metadata_t {
+  row_t row0;
+  row_t row1;
+  col_t col;
+  bitvec_hdr bvh0;
+  bitvec_hdr bvh1;
+};
+
+struct str {
+  bit<8> a;
+  bit<8> b;
+};
+
+header parsed_packet_t {
+  str s;
+  bit<16> c;
+};
+
+parser parse(packet_in pk, out parsed_packet_t h,
+             inout local_metadata_t local_metadata,
+             inout standard_metadata_t standard_metadata) {
+  state start {
+    transition accept;
+  }
+}
+
+control ingress(inout parsed_packet_t h,
+                inout local_metadata_t local_metadata,
+	        inout standard_metadata_t standard_metadata) {        
+    apply {
```

Comment:
your indentation is inconsistent - choose either 4 or 2 spaces.

Reply:
Chose 4 and fixed this file.